docs/source/whatsnew.rst

```diff
@@ -138,6 +135,33 @@ To learn more about the new private data patterns, see :doc:`private-data/privat
 for information about private data sharing and :doc:`private-data-arch` (reference information)
 for documentation about private data collection configuration and implicit collections.
 
+External chaincode launcher
+---------------------------
+
+While chaincode is still run in a docker container by default in Fabric v2.0,
+the external chaincode launcher feature empowers operators to build and launch
+chaincode with the technology of their choice.
+
+* **Eliminate Docker daemon dependency** Prior releases of Fabric required
+  peers to have access to a Docker daemon in order to build and launch
+  chaincode - something that may not be desirable in production environments
+  due to the privileges required by the peer process.
+
+* **Alternatives to containers** Chaincode is no longer required to be run
+  in Docker containers, and may be executed in the operator’s choice of
+  environment (including containers).
+
+* **External builder executables** An operator can provide a set of external
+  builder executables to override how the peer builds and launches chaincode.
+
+* **Chaincode as an external service** Traditionally, chaincodes are launched
+  by and then connect to the peer. It is now possible to run chaincode as
```

Comment:
nit: This first sentence sounds a bit weird. Maybe `Traditionally, chaincodes are launched by and then connected to the peer.`?

Reply:
+1

Reply:
Changing it to "Traditionally, chaincodes are launched by the peer, and then connect back to the peer."